pkg/cloud/provider/provider.go

```diff
@@ -291,41 +299,51 @@ func getClusterProperties(node *clustercache.Node) clusterProperties {
 	if env.IsUseCustomProvider() {
 		// Use CSV provider if set
 		if env.IsUseCSVProvider() {
+			log.Debug("using custom CSV provider")
```

Comment:
It was nigh on impossible to determine what provider was in use by a given deployment, hence these new debug logs.